go-controller/pkg/node/gateway_init.go

```diff
@@ -15,36 +15,17 @@ import (
 	util "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
 )
 
-// bridgedGatewayNodeSetup makes the bridge's MAC address permanent (if needed), sets up
-// the physical network name mappings for the bridge, and returns an ifaceID
+// bridgedGatewayNodeSetup sets up the physical network name mappings for the bridge, and returns an ifaceID
 // created from the bridge name and the node name
-func bridgedGatewayNodeSetup(nodeName, bridgeName, bridgeInterface, physicalNetworkName string,
-	syncBridgeMAC bool) (string, net.HardwareAddr, error) {
-	// A OVS bridge's mac address can change when ports are added to it.
-	// We cannot let that happen, so make the bridge mac address permanent.
-	macAddress, err := util.GetOVSPortMACAddress(bridgeInterface)
-	if err != nil {
-		return "", nil, err
-	}
-	if syncBridgeMAC {
```

Comment:
I wonder why @girishmg had made this conditional before: 

https://github.com/ovn-org/ovn-kubernetes/commit/8af2a6f7d5d6716b1e7c1ee9f4ccbe80c5a6935a#diff-e985c41826ef1fb0d6ba15fb3aa7368513d254b58bf0ce7221ad9ffb718c1bc4R14